src/meson.build

```diff
@@ -15,7 +15,7 @@ symbol_export_define = ['-DGST_EXPORTS']
 
 credentials = library('gstawscredentials-@0@'.format(apiversion),
   ['gstawscredentials.cpp'],
-  dependencies : [aws_cpp_sdk_sts_dep, gst_dep],
+  dependencies : [aws_cpp_sdk_sts_dep, gst_dep, aws_c_common_dep, aws_crt_cpp_dep],
```

Comment:
just out of curiosity; why is that needed? isn't it an implicit dependency of `aws-cpp-sdk-sts`?

Reply:
I’ve spoken with the engineer who requested this, and they advised it's because they're building with yocto for embedded. They tried to rebuild without the dependencies, and it still fails. For context this is where they have their [patch](https://github.com/aws4embeddedlinux/meta-aws/blob/master-next/recipes-multimedia/amazon-s3-gst-plugin/files/fix-linking-of-aws-crt-cpp.patch) in their repo.

Reply:
We have an open issue with the AWS SDK to investigate this see https://github.com/aws/aws-sdk-cpp/issues/2643